app/[lang]/chapters/[slug]/[lesson]/page.tsx

```diff
@@ -108,18 +108,6 @@ export default function Page({ params }) {
     )
   }
 
-  // If account and progress data have been loaded, but there is no progress obj
-  if (!account || !progress) {
-    return (
-      <>
-        <Head />
-        <div className="flex h-full w-full grow flex-col items-center justify-center">
-          <SignIn lang={params.lang} />
-        </div>
-      </>
-    )
-  }
-
```

Comment:
Looking back at this I now see what Jonas was saying, didn't we already remove this code? how did it end up back in the codebase from when it was removed in #399

Reply:
This was probably a result of us having 6+ PR's pending for a long time all touching various parts of the codebase. @shaavan can you make sure this PR removes this section like you did in #399 again?

Reply:
Addressed in b61b1001da3a6ae75a664201d422e40e1384bb32

Thanks!